src/Raven.Server/Indexing/LuceneVoronDirectory.cs

```diff
@@ -68,19 +68,18 @@ public override void TouchFile(string name)
             if (readResult == null)
                 throw new FileNotFoundException("Could not find file", name);
 
-            filesTree.Add(name, Stream.Null);
+            var pos = filesTree.DirectAdd(name, readResult.Reader.Length);
```

Comment:
Note that this is on "Files" tree, and the value here is just the size of a file. For actual data we have a tree per file where values are chunks.